app/src/main/java/woowacourse/payments/ui/newcard/component/SelectedBankRow.kt

```diff
@@ -0,0 +1,49 @@
+package woowacourse.payments.ui.newcard.component
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.ExperimentalLayoutApi
+import androidx.compose.foundation.layout.FlowRow
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import woowacourse.payments.ui.newcard.uiModel.CardCompanyUiModel
+import woowacourse.payments.ui.newcard.uiModel.cardCompanyMap
+
+private const val COLUMN_COUNT = 4
+
+@OptIn(ExperimentalLayoutApi::class)
+@Composable
+fun SelectedBankRow(
+    selectedBank: (CardCompanyUiModel) -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    FlowRow(
+        modifier = modifier
+            .padding(43.dp)
+            .height(227.dp)
+            .fillMaxWidth(),
+        horizontalArrangement = Arrangement.SpaceBetween,
+        verticalArrangement = Arrangement.SpaceEvenly,
+        maxItemsInEachRow = COLUMN_COUNT,
+    ) {
+        repeat(cardCompanyMap.entries.filterNot { it == CardCompanyUiModel.Default() }.size) { index: Int ->
+            if (selectedBank != CardCompanyUiModel.Default()) {
+                val cardCompanyType = cardCompanyMap.entries.toList()[index].value
+                SelectableCardCompanyLogo(
+                    cardCompany = cardCompanyType,
+                    selectedBank = { selectedBank -> selectedBank(selectedBank) })
+            }
+        }
```

Comment:
```suggestion
        cardCompanyMap.entries.filterNot { it == CardCompanyUiModel.Default() }.forEach {
            SelectableCardCompanyLogo(
                cardCompany = it.value,
                selectedBank = { selectedBank -> selectedBank(selectedBank) })
        }
```
ROW 상수가 제거되어 FlowRow의 의도대로 사용하기 시작 한 것 같네요 👍👍

Default를 두번 체크하면서 index로 접근하는 부분은 제안한 방식으로 바꿔보는건 어떨지 제안 남겨봅니당!